storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -735,39 +756,108 @@ private static int numUsedWorkers(SchedulerAssignment assignment) {
         return ret;
     }
 
-    private static Map<String, Map<List<Long>, List<Object>>> computeNewTopoToExecToNodePort(
-        Map<String, SchedulerAssignment> schedAssignments, Map<String, Assignment> existingAssignments) {
-        Map<String, Map<List<Long>, List<Object>>> ret = computeTopoToExecToNodePort(schedAssignments);
-        // Print some useful information
-        if (existingAssignments != null && !existingAssignments.isEmpty()) {
-            for (Entry<String, Map<List<Long>, List<Object>>> entry : ret.entrySet()) {
-                String topoId = entry.getKey();
-                Map<List<Long>, List<Object>> execToNodePort = entry.getValue();
-                Assignment assignment = existingAssignments.get(topoId);
-                if (assignment == null) {
-                    continue;
+    private boolean inspectSchduling(Map<String, Assignment> existingAssignments,
```

Comment:
The name is a little vague. How about `calculateAssignmentChanged` or something like that? Also there's a missing e in scheduling.

Reply:
I don't think we should use `calculateAssignmentChanged` as it doesn't really calculate anything for scheduling process so much as inspect/audit the throughput of scheduling. I would prefer naming `auditAssignmentChanges` or `reviewAssignmentChanges`, etc.

Reply:
Sure, sounds good.